src/main/java/at/hannibal2/skyhanni/config/ConfigManager.kt

```diff
@@ -239,7 +239,13 @@ class ConfigManager {
                 findPositionLinks(field.get(obj), slog)
                 continue
             }
-            val configLink = field.getAnnotation(ConfigLink::class.java) ?: continue
+            val configLink = field.getAnnotation(ConfigLink::class.java)
+            if (configLink == null) {
+                val abc = "${field.declaringClass.name}.${field.name}"
+                val def = abc.replace("at.hannibal2.skyhanni.config.", "")
+                println("WEE WOO WEE WOO HIER FEHLT EIN @CONFIGLINK: $def")
```

Comment:
maybe the user should get an error message for this so we can actually fix them

Reply:
Sadly, we cant.
We have a bunch of gui elements that dont have a config setting. they are all enabled via commands:
```
features.garden.GardenConfig.cropSpeedMeterPos
features.misc.MiscConfig.collectionCounterPos
features.misc.MiscConfig.lockedMouseDisplay
features.dev.DebugConfig.trackSoundPosition
features.dev.DevConfig.debugPos
features.dev.DevConfig.debugLocationPos
features.dev.DevConfig.debugItemPos
```

Reply:
i see